File: primerTrimestre/java basico/1secuenciales/Ej14sec.java
/**Dado un numero de dos cifras, diseñe un algoritmo que permita obtener el numero invertido.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej14sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    int decenas;
    int unidades;
    int invertido;
    int num;
    
    //Pedimos un numero de dos cifras.
    System.out.println("Inserte un numero de dos cifras: ");
    num = s.nextInt();
    
    
    decenas = num/10; //Division entera del numero entre 10.
    unidades = num%10; //El restante del numero al dividir entre 10.
    invertido = unidades*10+decenas;
    
    //Mostramos las decenas y unidades invertidas.
    System.out.println("El numero invertido es "+invertido);
    
  }

}

File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej14sec.py
# Programa: ej14sec.py
# Propósito: Dado un número de dos cifras, diseñe un algoritmo que permita obtener el número invertido.
#
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * decenas
#   * unidades
#   * num
#   * invertido
#
# Algoritmo:
#   LEER decenas, unidades, num
#   decenas <-- num/10
#   unidades <-- num%10
#   invertido = 10*unidades+decenas
#   ESCRIBIR invertido


# Petición de datos
num = int(input("Inserte un número de dos cifras: "))
print("---------------------------------------------------------------")

#Cálculos
decenas = num/10
unidades = num%10
invertido = unidades*10+decenas

#Cálculos y Salida. num[::-1]
print ("El número invertido es", invertido)





	
	

	






	
	

	
